### ISSUE TITLE
[Bug]: Popper components throwing error on SSR

### ISSUE THREAD

**Issue by MahediAmin** [radix-vue/radix-vue#408]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Shadcn Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://www.shadcn-vue.com/docs/components/tooltip.html

### Steps to reproduce

<script setup>

import {
  Tooltip,
  TooltipContent,
  TooltipProvider,
  TooltipTrigger,
} from "@/components/ui/tooltip";
</script>
<template>
  <TooltipProvider>
    <Tooltip>
      <TooltipTrigger>Hover</TooltipTrigger>
      <TooltipContent>
        <p>Add to library</p>
      </TooltipContent>
    </Tooltip>
  </TooltipProvider>
</template>

<style lang=""></style>

### Describe the bug

when i  use tooltip into my nuxt project 
<img width="602" alt="image" src="https://github.com/radix-vue/shadcn-vue/assets/26061598/846108fd-1f9b-4223-a0e1-f63141fa0616">
get this error.

if i use redix-tooltip directly there is no error

**Comment by BayBreezy** [comment]
> @BayBreezy could you point me to which triggers are causing similar issue? I updated a few.. and will look into the real problem today

@zernonia , I removed the client-only tag from all the Radix-vue components and I no longer get the warning/error.
Thanks for fixing this. 
If anything comes up, I will be sure to let you know

**Comment by zernonia** [comment]
Thanks for that @BayBreezy ! 😁

**Comment by BayBreezy** [comment]
Hey @zernonia , I am getting the error with the Tooltip trigger component. If not wrapped in the client-only tag, I get all sorts of hydration warnings and it causes the child of the trigger to be duplicated.

**Comment by zernonia** [comment]
hmmm.. @BayBreezy  Im not seeing that.. Could you provide some screenshot or minimal repro?

**Comment by BayBreezy** [comment]
Sure @zernonia , here is the stackblitz: https://stackblitz.com/edit/nuxt-starter-ws3qeq?file=pages%2Findex.vue

I added 2 buttons that are triggers for tooltips. Check the console and you will see the issue with the one not wrapped in the client-only tag

**Comment by zernonia** [comment]
@BayBreezy  you should as `asChild` props to trigger when you use another `button`.

eg: wrapping native button inside another button would render hydration error by default.
```
    <button>
        <button>Test</button>
    </button>
```

So, in your code, add `asChild` as such

```
       <UITooltipTrigger asChild>
              <UIButton variant="outline">
                <Icon class="h-5 w-5" name="solar:moon-fog-bold-duotone" /> Does
                not work on init load
              </UIButton>
            </UITooltipTrigger>
```

**Comment by BayBreezy** [comment]
Ohhhhhhh. My fault

**Comment by zernonia** [comment]
No problem. You are welcome! 😁

**Comment by BayBreezy** [comment]
@zernonia , So adding asChild removed the hydration error, but I still get this error when I hover over it.
`Uncaught (in promise) TypeError: n.getBoundingClientRect is not a function`

<img width="883" alt="image" src="https://github.com/radix-vue/radix-vue/assets/53905713/c6796973-aac5-478b-8d54-a007d065e74d">

**Comment by zernonia** [comment]
@BayBreezy  the fixes is in `0.2.3`

![image](https://github.com/radix-vue/radix-vue/assets/59365435/8b7cc7e3-8af9-4169-be2e-b0fca4aab65e)

**Comment by BayBreezy** [comment]
Oh.. The website is saying that 0.2.2 is the latest. Another oversight on my part. Thank you. It is working as intended now.
